app/containers/Forms/SummaryFormFieldTemplate.tsx

```diff
@@ -118,9 +118,6 @@ const FormFieldTemplate: React.FunctionComponent<FieldTemplateProps> = ({
           padding-top: 0;
           padding-bottom: 50px;
         }
-        .summary-card .card-header h2 {
-          font-size: 30px;
-        }
```

Comment:
This was polluting styles located just outside the scope of this form field component. It's effectively been moved to the correct location.